driver/drivers.go

```diff
@@ -161,9 +161,9 @@ func (d *Drivers) Delete(taskName string) error {
 	}
 
 	// delete driver templates associated with task
-	for val, id := range d.driverTemplates {
-		if val == taskName {
-			delete(d.driverTemplates, id)
+	for k, v := range d.driverTemplates {
+		if v == taskName {
+			delete(d.driverTemplates, k)
```

Comment:
Is there any potential issue that will occur from modifying the map you're iterating over? Not sure how that works in go.

Reply:
yep, this is okay, and idiomatic as well: 

https://go.dev/doc/effective_go#for (see If you only need the first item in the range (the key or index), drop the second)